ohm-js/test/test-ohm-syntax.js

```diff
@@ -976,6 +976,47 @@ test('inheritance', t => {
   t.end();
 });
 
+test('override with "..."', t => {
+  let g = ohm.grammar('G { letter := "@" | ... }');
+  t.equal(g.match('@', 'letter').succeeded(), true);
+  t.equal(g.match('a', 'letter').succeeded(), true);
```

Comment:
We should add another test that ensures that the "before terms" get evaluated before the "...". Something like this might do the trick:

```
let g = ohm.grammar('G { letter := "" | ... }');
t.equal(g.match('', 'letter').succeeded(), true);
t.equal(g.match('a', 'letter').succeeded(), false);
```

Reply:
Nice! Added.